dm/syncer/syncer.go

```diff
@@ -1054,18 +1144,19 @@ func (s *Syncer) resetShardingGroup(table *filter.Table) {
 //
 // we may need to refactor the concurrency model to make the work-flow more clearer later.
 func (s *Syncer) flushCheckPoints() error {
```

Comment:
it's no harm to put a duplicated s.wg.Wait() inside this function, to make some logic clear 😂